### PR TITLE
Adding a static menu and necessary components

### PR DESCRIPTION
**Pull request by SaurabhJamadagni**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
- [x] I have updated the [CHANGELOG](https://github.com/scribe-org/Scribe-iOS/blob/main/CHANGELOG.md) with a description of my changes for the upcoming release (if necessary) 

---

### Description

Hey @andrewtavis! Like I mentioned in our meeting last week, sorry for this clutter of a pull request. I promise the next pull requests will be atomic and specific in nature. I was extremely rusty with UIKit and needed a lot of refresher articles which really clumped the commits together beyond resolve. 

- This pull request adds a Tab bar navigation to the Scribe app. We can toggle between Installation, Settings, About sections.
- The Installation tab is just the older Scribe app screen. Will make changes to it as I get to their respective child issues.
- The Settings and About tab use a nested UITableView approach. They have their separate files to interpolate the tables on these screens.
  - A few of the older commits will basically show the older approach I was attempting where each section was a separate UITableView.
  - It required having multiple IBOutlet variables and also needed a ScrollView to be added manually. (Adding constraints to the ScrollView was a mess)
  - Nested table views allow us to have only a single table view and also helps with as the content scales or screen size reduces.
  - The custom cell components can be found in the `Components` folder.
- Necessary icons have been added to the Assets catalogue along with their dark mode versions.

Tested on devices iPhone 14 Pro, iPhone SE (3rd gen)

### Related issue

- #16 
- #320 
- #309

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks @SaurabhJamadagni! I'll give this a look in the coming days and let you know if we're good to go 😊 GSoC! ☀️

File: Scribe/AboutTab/AboutTableData.swift
Comment (wkyoshida) [inline Scribe/AboutTab/AboutTableData.swift:18]: Hey hey :wave: 

Would texts like these, the `sectionTitle`s here, be good candidates for i18n? I see some support for it started already under [`./Scribe/AppTexts`](https://github.com/scribe-org/Scribe-iOS/tree/main/Scribe/AppTexts). Thinking that the app texts here could be good ways as well to continue working i18n into Scribe.

---
Implementing via the below perhaps?
- https://developer.apple.com/documentation/swiftui/preparing-views-for-localization
- https://developer.apple.com/documentation/swiftui/localizedstringkey

**Comment by andrewtavis** [comment]
@SaurabhJamadagni, as discussed in the weekly:

- [ ] Row elements should have an increased height
- [ ] There should be more padding between the icon and text for table elements
- [ ] Page headers should be inline with the tables (table headers should be indented a little bit as they are already)

**Comment by SaurabhJamadagni** [comment]
Commit a25c0bf716bb9cb45e766dab1fa84c47b6c9cfaa should do it. Let me know @andrewtavis if everything is in order. Thanks!

**Comment by SaurabhJamadagni** [comment]
Hey @andrewtavis, just a small git related question. If I push my branch changes to remote, they'll be appended in this pull request itself right? So I should I push the changes to the same request or wait so that issue #315 has it's own separate pull request after this one is merged?

**Comment by andrewtavis** [comment]
I have one more activist PR I want to get to and will then jump over here. Hopefully tonight, but if not then tomorrow!

**Comment by SaurabhJamadagni** [comment]
No worries @andrewtavis, I'll wait for this PR to be merged. In case any new contributors wish to see how a certain issue was resolved or implemented they might have to come to this giant combined PR otherwise. Might get overwhelming 😅

**Comment by andrewtavis** [comment]
Sounds good, @SaurabhJamadagni! As I said I’ll try to get to it tonight :)

**Review by andrewtavis — APPROVED** [review]
Minor comments on this, @SaurabhJamadagni (and of course sorry for getting to this late - GF's dad has been in town and I've been helping them out):

- Feel free to remove the `View privacy policy` button, but then I'm assuming your plan is to do that in #317 :)
- I'll go through and do some minor code formatting for spaces between `}`s
- Structure is really solid and I really appreciate the thought that you put into it!
- I'll doubtless play around with it a bit in the coming weeks and make some minor minor changes as I go, but I'm really happy with this 😊 Crazy to see Scribe like this on my computer, and even crazier to think of what's to come and actually having this on devices! Thank you! 🎉

**Comment by andrewtavis** [comment]
Closing #16, #320 and #309 😊🥳

**Comment by SaurabhJamadagni** [comment]
Hey @andrewtavis! No problem with the delay, it was nothing :)

Just got back yesterday evening from the hackathon I told you last week. Thank you so much for the review. Completely with you on the progress on Scribe. It is super exciting to be able to contribute to it!! Thanks for taking care of any formatting that I might have missed. See you tonight for our check-in 😊

**Comment by andrewtavis** [comment]
Looking forward to hearing about the hackathon later!

**Pull request merged**
